std/python/Lib.hx

```diff
@@ -36,7 +36,7 @@ class Lib {
 	/**
 		Print the specified value on the default output.
 	**/
-	public static function print(v:Dynamic):Void {
+	public static function print_(v:Dynamic):Void {
```

Comment:
It does matter, because haxe-generated code can be used as a library by thrid-party native app.